### PR TITLE
Rename Acl to ACL

### PR DESCRIPTION
**Pull request by idafurjes**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by gotjosh — APPROVED** [review]
LGTM

Thanks!

**Review by eleijonmarck — APPROVED** [review]
LGTM. Searched through and seems good

**Review by sakjur — COMMENTED** [review]
I saw that you fixed the Xorm tags while I was reviewing, but I think the JSON tags still needs to be fixed?

File: pkg/models/folders.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ type Folder struct {
 
 	UpdatedBy int64
 	CreatedBy int64
-	HasAcl    bool
+	HasACL    bool
```
[the remark this answers is not in the document]
Comment (sakjur) [reply pkg/models/folders.go:20]: (is this ever used as a database model?)

**Review by sakjur — APPROVED** [review]
Turns out you fixed pretty much all of my feedback before I gave it 👍

Only remaining q is whether we need a Xorm tag for the folder or not 🙂

File: pkg/models/folders.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (idafurjes) [reply pkg/models/folders.go:20]: I changed all of them, I can change it back, I don't have a strong opinion on this one

**Pull request merged**